rules/windows/registry_event/win_outlook_registry_TodayPage.yml

```diff
@@ -0,0 +1,34 @@
+title: Persistent Outlook Landing Pages
+id: 487bb375-12ef-41f6-baae-c6a1572b4dd1
+description: Detects the manipulation of persistant URLs which could execute malicious code
+status: experimental
+references:
+  - https://speakerdeck.com/heirhabarov/hunting-for-persistence-via-microsoft-exchange-server-or-outlook?slide=70
+author: Tobias Michalski
+date: 2021/06/10
+tags:
+  - attack.persistence
+  - attack.t1112
+logsource:
+  product: windows
+  category: registry_event
+detection:
+  selection1:
+    EventID:
```

Comment:
Don't use EventIDs in rules written in generic categories. The EventID gets applied during rule conversion with an appropriate config. Simply remove the EventID elements.